### ISSUE TITLE
X11 requirements

### ISSUE THREAD

**Issue by bbqsrc** [divvun/kbdgen-rs#2]
- [ ] Dead keys are generated but transforms are not?
- [ ] loading `se/se` makes a keyboard that is predominantly nulls, and probably should not
- [ ] use `pyo3` so we can run the convertor from Python (kbdgen will own this library)

**Comment by killercup** [comment]
> loading se/se makes a keyboard that is predominantly nulls, and probably should not

Yeah, it just took me an embarrassingly long time to figure out that -- from what I can now tell -- you are not supposed to use `se(se)`, but `se(basic)`. The latter is literally just `include "latin(type2)"` and `include "se(se)"`, and `se(se)` overwrites the keys specific for Swedish keyboards. I assume this is so that they can re-use the `se(se)` definitions in other derivations.

Unfortunately, users cannot easily tell this. For example, the modifiers for `se(se)` and `se(nodeadkeys)` are the same, so a heuristic that would annotate one as "internal" would also flag the other one.
